Helpers/PaymentHelper.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Alma\MonthlyPayments\Helpers;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+Use InvalidArgumentException;
+
+class PaymentHelper extends AbstractHelper
+{
+    CONST NO_ORDER_ID = 'No order_id in alma Payment';
+    CONST PAYMENT_ORDER_ID_KEY = 'order_id';
+
+    public function getOrderIdFromAlmaPayment($almaPayment):string
```

Comment:
```suggestion
    /**
     * @param Payment $almaPayment
     * @return string
     */
    public function getOrderIdFromAlmaPayment(Payment $almaPayment):string
```

phpdoc + typehint of `$almaPayment`

Reply:
Done